extra/1-radio-stations.js

```diff
@@ -25,7 +31,16 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
-
+function getStations() {
```

Comment:
This looks good too!

Another tip, you can try to use `filter` to implement this

Reply:
@qingwei91 Still cant figure it out how to not lopp.
Maybe I didn't understand your meaning.
Can you help me with that please? 😊
Thanks

Reply:
Hi @momahboobian , you can read about this method, then it should be clearer: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter